src/api/RestAPI.ts

```diff
@@ -186,11 +208,9 @@ export class RestAPI extends EventEmitter {
         uri: string,
         data?: SendData<Data>,
         auth?: AuthParams,
-        callback?: ResponseCallback<ResponseBody>,
-        requireAuth = true,
-        acceptType?: string,
-        keyFormatter = toSnakeCase
+        options?: ApiSendOptions
     ): Promise<ResponseBody | string | Blob | FormData> {
+        const { requireAuth = true, acceptType, keyFormatter = toSnakeCase } = options || {};
```

Comment:
`||` -> `??` 😅
or maybe move this up to be a default parameter value? `options?: ApiSendOptions = {}`